src/index.css

```diff
@@ -0,0 +1,68 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700;800&display=swap');
+
+body {
+  margin: 0;
+  font-family: 'Montserrat', sans-serif;
+  background: #f5f6fa;
+}
+
+.wrapper {
+  padding: 4.3rem 3.75rem 7.688rem 6.25rem;
+}
+
+.panel-bg {
+  background: #fafafa;
+  border-radius: 4px;
+
+  /* height: 68.563rem; */
```

Comment:
- [ ] [OPTIONAL] Kindly make sure there is no commented code in your codebase so it looks clean and easy to navigate